dependencies.rosinstall

```diff
@@ -10,6 +10,12 @@
     uri: 'https://github.com/ros-industrial/noether.git'
     version: 9c97ac493f9eb2cee83523fadf5cf331f2c22ceb
 
+# heat_raster
+- git:
+    local-name: heat_raster
+    uri: 'https://github.com/ros-industrial/heat_raster.git'

```

Comment:
I believe this is the issue, it should be swri-robotics.

Reply:
Thanks. I'm slow sometimes

Reply:
I made the change, lets see if CI now runs.

Reply:
I update the git hash in the rosinstall because it was failing to find suitesparse which solve this issue but now there are build errors related to message definition changes.

Reply:
done, and has a selected commit that compiles